qiita_db/test/test_data.py

```diff
@@ -261,6 +261,28 @@ def test_remove_filepath_errors(self):
         with self.assertRaises(QiitaDBError):
             RawData(2).remove_filepath(fp)
 
+    def test_exists(self):
+        self.assertTrue(RawData.exists(1))
+        self.assertFalse(RawData.exists(1000))
+
+    def test_delete(self):
```

Comment:
Just a comment, not necessary to change, but this test case could probably be divided into cases that raise an exception and cases that don't.